src/share/cxx/cxx_f90_interface.cpp

```diff
@@ -100,27 +100,67 @@ void init_hvcoord_c (const Real& ps0, CRCPtr& hybrid_am_ptr, CRCPtr& hybrid_ai_p
   hvcoord.init(ps0,hybrid_am_ptr,hybrid_ai_ptr,hybrid_bm_ptr,hybrid_bi_ptr);
 }
 
-void cxx_push_results_to_f90(F90Ptr& elem_state_v_ptr,   F90Ptr& elem_state_temp_ptr, F90Ptr& elem_state_dp3d_ptr,
```

Comment:
If you remove this interface, is it stil possible to run homme standalone?

Reply:
If the answer is 'no', can we split the push in 'results' and 'forcing'?

Reply:
Ah, you're right; it's currently crashing probably because of this